tests/RoutingTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Tests;
+
+use Generator;
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+class RoutingTest extends WebTestCase
+{
+    /**
+     * @dataProvider urlProvider
+     */
+    public function testRouteIsSuccessful(string $url): void
+    {
+        $client = static::createClient();
+        $client->request('GET', $url);
+
+        $this->assertResponseIsSuccessful();
+    }
+
+    public function urlProvider(): mixed
+    {
+        yield ['/'];
+        yield ['/video/show/1'];
```

Comment:
Not sure this test will succeed

Reply:
It will succeed if the test database video ids start with 1.